SUMMARY.md

```diff
@@ -71,11 +74,29 @@
   * [Working with cursors](tutorials/advanced-development/database/cursors.md)
```

Comment:
Not a pieces topic. Move it up.

Reply:
These are within the "The Database" section. Should they still move elsewhere?